### ISSUE TITLE
VA variable for Krait {TXT:Ship model (spoken)} reads phonetic code

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#1574]
## EDDI version in which issue is found
v3.4.3 release

## VoiceAttack version in which issue is found (as applicable)
v1.7.8 release

## Steps to reproduce
In VoiceAttack use {TXT: Ship model (spoken)} for both Krait models.

## Expected
VA/EDDI says "Krait Phantom" or "Krait Mark 2"

## Observed
VA/EDDI reads out the programming(?) code for the phonetic version of the model.

## Investigation
I am opening this as I was asked about this happening to another user, Mj_Alvega.  He wasn't sure how to make an error report, so I'm doing this on his behalf, as well as my own now.

Alvega asked me about a command he has, where it makes use of {TXT:Ship model (spoken)} and {TXT:Ship name} and randomly chooses between the two.  He told me that for all his ships, the model is said fine, except for his Krait MkII, where it was reading things like "equals" and "slash" (his words) and sounded very wrong.

He asked me to investigate, so I had him send his VA profile over to me to test and look over.  I am currently in my Krait Phantom, so I tried it with that.  I do seem to hear "slash" and "equals", and I can hear it mention "phoneme" in amongst a lot of other characters, and buried within them, "Krait".

So, I made up a command that copies the output of {TXT:Ship model (spoken)} to the VA log window.  This is what gets spoken by VA:-
`<phoneme alphabet="ipa" ph="ˈkreɪt">Krait</phoneme>`
All the "less than" and "greater than" and "slash" are spoken literally as words.  The line is read out exactly as it's written.  I'm fairly sure it's not supposed to do that.

However, I am out in the middle of nowhere still, so I don't have access to other ships to try them and see what I get.  If I remove the (spoken) part, then the model is said as "Krait Phantom", as I would expect.

I'm sorry that some of this is second hand reporting, but if you need any further info, just let me (or Mj_Alvega on the forums) know.

**Comment by Tkael** [comment]
Let's see how the actual Cottle script is written. Please share that?

**Comment by Tkael** [comment]
In VoiceAttack, I believe there should be an option to use SSML. Try turning that on.

**Comment by Darkcyde13** [comment]
It took me a while to find the option, it's hidden away in the actual 'Say' command window options.

I enabled it, but then the command was totally silent.  After bit of Googling, I found that I had to enter a code block to make it work:-
```
<?xml version="1.0"?>
<speak version="1.0" xmlns="http://www.w3.org/2001/10/synthesis"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.w3.org/2001/10/synthesis
                   http://www.w3.org/TR/speech-synthesis/synthesis.xsd"
         xml:lang="en-UK">
{TXT:Ship model (spoken)}.
</speak>
```
Like this, it now says "Krait", but doesn't specify the model of Krait, which makes sense as the previous phoneme code shows only "Krait".

Is this how it's supposed to work then?  Do we need to add all that SSML code into every 'Say' command?  If so, how does Alvega's other ships models get spoken ok without it, or the SSML switch?

**Comment by Tkael** [comment]
I'd forgotten that VoiceAttack required the ssml to be wrapped in the full header and <speak> command... in Cottle the content of the variable would have sufficed.

In the context of what you're describing, I'm not sure that `{TXT:Ship model (spoken)}` is the best choice, but if so then you're looking at constructing something like the above to use it.

**Comment by Tkael** [comment]
It's hard for me to comment on how it is or isn't working for his other ships, but IPA isn't necessary for rendering every ship name so maybe his other ships have simpler names that don't require IPA?

**Comment by Darkcyde13** [comment]
Maybe, yeah.  I know he's currently out mining in his Python, and as far I know he meant that that one was ok.

I did tell him that by removing (spoken), I had got it to work as expected, with it saying "Krait Phantom" for me.  He said he would try removing it and see what happens.  He's Portuguese living in South America at the moment, so I think his language is set to pt-BR, which may mean that (spoken) made it sound better for him.

The thing is, he has merged an old version of my profile with another he found, some time ago, and has been tinkering with it ever since.  He has admitted that he doesn't really know a great deal about VA or EDDI coding, so I can assume most of what he has is just copy/paste from various sources.

I wish I was back in civilised space so I could try more things for myself, but I guess I'll have to wait to see what happens when he tries out removing (spoken).

Thanks for your input anyway. :)
